app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
```

Comment:
Don't we need has_many :bookings as, well?

Reply:
Yes, but we don't have a booking model yet, it's better to add it later

Reply:
that's true 👍